protocols/kad/src/behaviour.rs

```diff
@@ -3338,3 +3396,9 @@ where
         .collect::<Vec<_>>()
         .join(", ")
 }
+
+#[derive(PartialEq)]
+pub enum InitialBootstrap {
```

Comment:
```suggestion
enum InitialBootstrap {
```

Let's make stuff private unless necessary.

Reply:
removed it